deploy/charts/rook-ceph/values.yaml

```diff
@@ -303,6 +341,12 @@ csi:
   csiAddons:
     enabled: false
     #image: "quay.io/csiaddons/k8s-sidecar:v0.2.1"
+  # Enable the nfs csi driver.
+  nfs:
+    enabled: false
+    #image: "mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0"
```

Comment:
Is there a design doc around installing the nfs csi driver from rook? I can see why we need a driver to go along with configuring of CephNFS, but here are a few questions:
- CephNFS is typically setup for external access to the cluster. Is there a scenario for mounting cephnfs inside the same cluster? Or is this really for external mode to connect to another cluster where cephnfs is configured? 
- Who owns the CSI driver? sig-storage?
- Do the owners recommend installing an instance of the driver specific for different solutions? We are installing the driver and registering it in the csi.ceph.com namespace. If we're not the owners, that seems unnatural at a glance.
- What is licensing for the csi driver?

Reply:
> Is there a design doc around installing the nfs csi driver from rook? I can see why we need a driver to go along with configuring of CephNFS, but here are a few questions:

Design doc at ceph-csi for CephNFS: https://github.com/ceph/ceph-csi/blob/devel/docs/design/proposals/nfs.md
I'll be adding cephNFS provisioner deployment as well to rook soon.

> 
> * CephNFS is typically setup for external access to the cluster. Is there a scenario for mounting cephnfs inside the same cluster? Or is this really for external mode to connect to another cluster where cephnfs is configured?

This daemonset will enable mount/unmount for CephNFS PVCs provisioned by Ceph-CSI similar to CephFS and RBD. 
I'll let @BlaineEXE @nixpanic comment on the external cluster part. 

> * Who owns the CSI driver? sig-storage?

It is under kubernetes-csi org.
https://github.com/kubernetes-csi/csi-driver-nfs

> * Do the owners recommend installing an instance of the driver specific for different solutions? We are installing the driver and registering it in the csi.ceph.com namespace. If we're not the owners, that seems unnatural at a glance.

Ceph-csi is getting capability to provisioner cephNFS PVC in this pr https://github.com/ceph/ceph-csi/pull/2948.
This will be added as `<ns>.nfs.csi.ceph.com` provisioner driver deployment.
We'll be using  [`csi-driver-nfs`](https://github.com/kubernetes-csi/csi-driver-nfs) as nodeplugin to mount/unmount 
CephNFS PVC as mentioned [here](https://github.com/ceph/ceph-csi/blob/devel/docs/design/proposals/nfs.md#node-plugin-for-mounting-nfs-exports).

With few [specific parameters ](https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/docs/driver-parameters.md#pvpvc-usage-static-provisioning) in PV spec, csi-driver-nfs will be able to mount CephNFS PVCs 

> * What is licensing for the csi driver?

[Apache License 2.0](https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/LICENSE)